OsmAnd/src/net/osmand/plus/OsmandSettings.java

```diff
@@ -3467,6 +3467,7 @@ protected boolean setValue(Object prefs, String val) {
 	}
 
 	public final CommonPreference<Float> ROUTE_RECALCULATION_DISTANCE = new FloatPreference("routing_recalc_distance", 0.f).makeProfile();
+	public final CommonPreference<Boolean> USE_ROUTE_RECALCULATION_DISTANCE = new BooleanPreference("use_routing_recalc_distance", false).makeProfile();
```

Comment:
USE_ROUTE_RECALCULATION_DISTANCE it should be reverted, new setting is not needed same as changes to RoutingHelper it only possibly breaks existing functionality.